docker-compose.yml

```diff
@@ -12,17 +12,21 @@ services:
     ports:
       - "19553:80"
       - "19552:8080"
-      - "19551:8082"
+      # - "19551:8082"
```

Comment:
any implication on commenting this?

Reply:
I didn't see any implication yet while performing tests on local playground 👀